e2e/libs/workload/workload.py

```diff
@@ -102,6 +102,33 @@ def write_pod_random_data(pod_name, size_in_mb, file_name,
             time.sleep(retry_interval)
 
 
+def write_pod_large_data(pod_name, size_in_gb, file_name,
+                          data_directory="/data", ):
+
+    wait_for_pod_status(pod_name, "Running")
+
+    retry_count, retry_interval = get_retry_count_and_interval()
+
+    for _ in range(retry_count):
+        try:
+            data_path = f"{data_directory}/{file_name}"
+            api = client.CoreV1Api()
+            write_data_cmd = [
+                '/bin/sh',
+                '-c',
+                f"fallocate -l {size_in_gb}G {data_path};\
```

Comment:
How long will the test case take after adopting `fallocate`?

Reply:
Reduced from approximately 13 hours to six hours.
![Screenshot_20241013_215127](https://github.com/user-attachments/assets/e246e5ae-49f6-4878-be75-be2b71541962)